src/xo-app/settings/remotes/index.js

```diff
@@ -23,34 +28,49 @@ const remoteTypes = {
 }
 
 class AbstractRemote extends Component {
+  _changeUrlElement = (value, element) => {
+    let { remote } = this.props
+    remote = {...remote}
```

Comment:
I think it would be best if `xo-web` 5 would use `xo-remote-parser` 3 to avoid a maintenance in a close future.